Sources/HtmlPrettyPrint/Pretty.swift

```diff
@@ -81,6 +84,17 @@ private func prettyPrint(attribute: Attribute) -> Doc {
       <> .text("\"")
   }
 
+  if attribute.key == "content" {
```

Comment:
`content` tags should prob also flow to multiple lines since they can get long, e.g. meta tags use this